### ISSUE TITLE
Cannot find the channel to turn on the lights

### ISSUE THREAD

**Issue by Angel-ESFM2001** [patrickelectric/bluerov_ros_playground#40]
Good morning. 

I tried to turn on the bluerov lights through some of the topics here https://github.com/patrickelectric/bluerov_ros_playground/blob/master/src/bridge/README.md but it does not work. Do you know a way to turn on the lights with ros?

**Comment by patrickelectric** [comment]
Hi @angelesfm have you done the bluerov setup ?
Do you have the lights configured in QGC ? Does it work via joystick ?

**Comment by Angel-ESFM2001** [comment]
Yes. I can turn on the lights with QGC and via joystick and I checked the RC input of this link https://www.ardusub.com/operators-manual/rc-input-and-output.html, I think that I need the channel 9 and 10 if I want to turn on the lights but in the list of topics doesn't have the channel 9 and 10.
